docs/CRSF_SENSORS.md

```diff
@@ -0,0 +1,40 @@
+# mLRS Documentation: CRSF Sensors #
+
+([back to main page](../README.md))
+
+([back to CRSF page](CRSF.md))
+
+mLRS provides the following telemetry sensors when using CRSF:
+
+Num | Datapoint | Description                                                              | Data Source |
+--- | --------- | ------------------------------------------------------------------------ | ----------- |
+1   | 1RSS       | Receiver RSSI                                                           | mLRS        |
+2   | 2RSS       | Not supported                                                           |             |
+3   | Alt        | Altitude                                                                | FC          |
+4   | ANT        | Not supported                                                           |             |
```

Comment:
I just pushed a commit to support ANT, it is the receiver seide receive antenna now

Reply:
Nice, added this.